daffodil-lib/src/test/scala/LineCounter.scala

```diff
@@ -116,20 +120,20 @@ trait LineCounterBase {
 
   def pairs: List[(String, Int)]
 
-  final def total = pairs.map { _._2 }.sum
+  final lazy val total = pairs.map { _._2 }.sum
 
   protected final def linesWithoutCopyright(f: File): List[String] = {
```

Comment:
This function no longer removes the copyright. Maybe it didn't work all that well? Seems like this is going to increase line numbers alot. Doesn't seem unreasonable, but just have to note that we shouldn't compare old numbers with new.